File: src/dev/api/ItemsUtil.js
let ItemsUtilJava = WRAP_JAVA("com.core.api.module.ItemsUtil");
let ItemsUtil = {
    api: WRAP_JAVA("com.core.api.mcpe.item.Item"),
    getItemById(id){
        return ItemsUtilJava.getItem(id);
    },
    overrideName:function(id,data, name){
        this.api.overrideName(id,data,name);
    },
    overrideArmorValue:function(id, value){
        this.api.overrideArmorValue(id, value);
    }
};

File: src/java/Boot/src/com/core/api/module/ItemsUtil.java
package com.core.api.module;

import com.core.api.engine.PointerClass;
import com.core.api.mcpe.item.Item;

public class ItemsUtil {
    native public static long getItemById(int id);

    public static PointerClass getItem(int id){
        return new Item(id);
    }
}
